packages/material/scss/scrollview/_theme.scss

```diff
@@ -0,0 +1,28 @@
+$scrollview-navigation-color: $base-text;
+
+@import "~@progress/kendo-theme-default/scss/scrollview/theme";
+
+@include exports("scrollview/theme/material") {
+    $scrollview-text-opacity: .4 !default;
+    $scrollview-bottom: $scrollview-pager-spacing-x * 2;
+
+    .k-scrollview-next:hover,
+    .k-scrollview-prev:hover {
+        &::before {
+            background-color: rgba($base-bg, $scrollview-text-opacity);
+        }
+    }
+
+    .k-scrollview-pageable {
+        bottom: $scrollview-bottom;
```

Comment:
This rule overrides [this hard-coded line](https://github.com/telerik/kendo-themes/blob/develop/packages/default/scss/scrollview/_layout.scss#L48) from theme-default. It would be great if we didn't have two rules (one from the default theme and one from the material theme) in the output CSS. To achieve this, you can introduce the `$scrollview-padding-*` variables in theme-default and take them into account. Then, override them before importing the `_layout` file.